tests/test_highlight.py

```diff
@@ -94,7 +94,7 @@ def test_highlight_diff(self):
 
     def test_no_highlight(self):
         filename = 'data'
-        source = 'It was the best of times, it was the worst of times, ...'
+        source = 'It was the best of times, it was the worst of times, ...\n'
```

Comment:
We changed the behavior to use the Text Lexer by default (instead of just returning the source).

I'm not actually sure if this is the best way to update the test - but it seemed like the cleanest way to keep testing what this was originally testing.